clipper_admin/clipper_manager.py

```diff
@@ -371,6 +368,58 @@ def register_application(self, name, model, input_type, default_output,
         print(r.text)
         return r.status_code == requests.codes.ok
 
+    def link_model_to_app(self, app_name, model_name):
+        """
+        Allows the model with `model_name` to be used by the app with `app_name`.
+        The model and app should both be registered with Clipper.
+
+        Parameters
+        ----------
+        app_name : str
+            The name of the application
+        model_name : str
+            The name of the model to link to the application
+
+        Returns
+        -------
+        bool
+            Returns true iff the model link request was successful
+        """
+        url = "http://%s:%d/admin/add_model_links" % (self.host,
+                                                      CLIPPER_MANAGEMENT_PORT)
+        req_json = json.dumps({
+            "app_name": app_name,
+            "model_names": [model_name]
+        })
+        headers = {'Content-type': 'application/json'}
+        r = requests.post(url, headers=headers, data=req_json)
```

Comment:
On second thought, let's re-add the call to `print(r.text)`. It's very helpful for user understanding - sorry!